docs/PC-8800.md

```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
 
-Put the bios files in lowercase otherwise the emulator won't find them, and result in black screen.
 
-## Controls
+## Emulator: [Quasi88](https://www.eonet.ne.jp/~showtime/quasi88/)
+
+Note that the emulator (and most documentation) is written in Japanese.
+
+This emulator (if available) can be installed from the experimental menu of the [RetroPie-Setup script.](Updating-RetroPie#using-the-retropie-setup-script)
+
+This emulator has better keyboard support. 
+
+Recommended for :
+
+- Working in "BASIC", "DOS" or "CP-M"
+- Editing disks or files
+
+## BIOS [Quasi88]
+
+Place your BIOS files in:
+```
+/home/pi/RetroPie/BIOS/pc88
+```
+
+## ROMS [Quasi88]
+
+Accepted File Extensions: **.d88 (disk) .88d (disk) .cmt (tape) .t88 (tape)** 
+
+Place your PC-88 ROMs in:
+```
+/home/pi/RetroPie/roms/pc88
+```
+
+## Controls [Quasi88]
+
 By default, Quasi88 uses the keyboard for controls.  To enable a controller:
 
 ```
-Press F12.  This will bring up a menu with several tabs at the top.
+Press "F12".  This will bring up a menu with several tabs at the top.
 Click the "mouse" tab.
 Click the dropdown that's labelled "nomouse". Change it to "joystick".
 ```
 
 Once this is done, the controls will be automatically mapped as such:
 ```
 Left Joystick - Number Pad
-A Button - Z key
-B button - X key
+A Button - "Z key"
+B button - "X key"
+```
+
+To quit the emulator:
+```
+Press 3 times "F12". 
 ```
 
 ## Emulator: [lr-quasi88](https://github.com/libretro/quasi88-libretro)
+
 This emulator can be installed from the experimental menu of the [RetroPie-Setup script.](Updating-RetroPie#using-the-retropie-setup-script)
 
+This emulator has better joystick support. 
+
+Recommended for :
+
+- Games
+
+
+## BIOS [lr-quasi88]
+
+Place your BIOS files in:
+```
+/home/pi/RetroPie/BIOS/quasi88
+```
+
+## ROMS [lr-quasi88]
+
+Accepted File Extensions: **.d88 (disk) .88d (disk) .cmt (tape) .t88 (tape) .m3u (playlist)** 
+
+Place your PC-88 ROMs in:
+```
+/home/pi/RetroPie/roms/pc88
+```
 
-## Controls
+## Controls [lr-quasi88]
 lr-quasi88 utilises Retroarch configurations.
 
 Add custom retroarch controls to the retroarch.cfg file in
 ```shell
 /opt/retropie/configs/pc88/retroarch.cfg
 ```
-For more information on custom RetroArch controls see: [RetroArch Configuration](RetroArch-Configuration)
\ No newline at end of file
+For more information on custom RetroArch controls see: [RetroArch Configuration](RetroArch-Configuration)
+
+
+## Insert disks in RetroArch menu (2 disk example) [lr-quasi88]:
+(you can also use the joystick in the RetroArch menu, if configured)
+```
+Start the emulator with the first disk
+Press "F1"
+Press "Z-key"
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Load 2-Disk Game"
+Select, with "X-key", the first disk file
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Load 2-Disk Game"
+Select, with "X-key", the second disk file
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Start 2-Disk Game"
+```
```

Comment:
Use a simple list to outline the steps, but don't use the keyboard shortcuts for operation - use the RetroPad conventions:
* Open the RGUI with _Hotkey + X_
* Go back (B button)
* Open the _Subsystems_ menu ...
..etc